src/db.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require("mongoose");
+
+const connectionString =
+  process.env.MONGO_CONNECTION_STRING ||
+  "mongodb://rootuser:examplepasswordikkO2Y@localhost:27017/flam";
+
+// Number of milliseconds to keep trying to establish a db connection
+const serverTimeout = process.env.MONGO_SERVER_TIMEOUT || 30000;
+// Number of milliseconds of inactivity after which to close sockets
+const socketTimeout = process.env.MONGO_SOCKET_TIMEOUT || 45000;
```

Comment:
I would suggest using connection string to configure everything that you can configure with connection string :) 
Here is the relevant section: https://www.mongodb.com/docs/manual/reference/connection-string/#timeout-options